src/include/distributed/shard_rebalancer.h

```diff
@@ -77,6 +77,8 @@
 #define REBALANCE_PROGRESS_MOVING 1
 #define REBALANCE_PROGRESS_MOVED 2
 
+#define MAX_PARALLEL_MOVES_PER_NODE 100
```

Comment:
Let's make this a nice round power of 2. This limit might be chosen based on number of cores, and those usually come in powers of 2.
```suggestion
#define MAX_PARALLEL_MOVES_PER_NODE 128
```